admin/data-collection.rst

```diff
@@ -916,18 +916,24 @@ seconds within which received offline data still relevant for threshold
 validation. By default it is set to 1 day.
```

Comment:
I would add information about supported  origins (sources) for agent cache in chapter about Agent caching mode